sherpa/astro/xspec/__init__.py

```diff
@@ -8146,7 +8149,7 @@ class XSzbabs(XSMultiplicativeModel):
 
     """
 
-    _calc =  _xspec.xszbabs
+    _calc =  model_wrapper("xszbabs")
```

Comment:
Why is `model_wrapper` used only on this model (excluding the optional `XSbtapec` model)?

Reply:
Because I needed a test for an "old" model. `model_wrapper` is not required for baseline models, i.e. models that are in the minimal version we support. I would expect us to keep adding models with the decorator. If we wanted to be braver, we would review the way we define xspec models... there is a lot of boilerplate that can be reduced. That would make the use of the `model_wrapper` implicit.